app/src/main/res/values-sw360dp/auto_dimens.xml

```diff
@@ -0,0 +1,22 @@
+<resources>
+    <dimen name="sp_23">23sp</dimen>
+    <dimen name="dp_200">10dp</dimen>
+    <dimen name="dp_10">10dp</dimen>
+    <dimen name="sp_13">13sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_13` appears to be unused
 Unused resources make applications larger and slow down builds.